### ISSUE TITLE
Wrong import of type Prisma in Flow Example

### ISSUE THREAD

**Issue by geroj** [prisma/prisma-examples#310]
I've found error while trying out your Flow graphql example

There is not correct import of type Prisma in file `types.js`
https://github.com/prisma/prisma-examples/blob/master/flow/graphql/src/types.js

There is work around which works for me
```javascript
/**
 * @flow
 */

import { Prisma } from './generated/prisma-client'

export type ExtractReturn<Fn> = $Call<<T>((...Iterable<any>) => T) => T, Fn>;

export interface Context {
  prisma: ExtractReturn<typeof Prisma>;
}
```

based on https://github.com/facebook/flow/issues/4002#issuecomment-384756198

Another option would be exposing PrismaInterface from https://github.com/prisma/prisma-examples/blob/a743b26362aa66d536a89d2a34b0cef5dbe646b1/flow/graphql/src/generated/prisma-client/index.js#L31

**Comment by nikolasburk** [comment]
Thanks for opening this issue. What exactly is the error you're seeing? I can't reproduce it at the moment, everything works fine when using the instructions from the README.

Can you please post the error message you're seeing and explain when exactly the error occurs?

**Comment by nikolasburk** [comment]
Thanks, do you want to submit a PR to fix this? Otherwise I can look into it later today :)

**Comment by geroj** [comment]
I think that it will be better when you will have look into this. My solution is just workaround for issue that generated types don't export `PrismaInterface`. Thank you :)

**Comment by divyenduz** [comment]
This is fixed in beta version `1.25.0-beta.4`. Please give it a go via `npm install -g prisma@beta` and let me know if that works for you. Thanks.
